src/main/scala/nl.knaw.dans.easy.solr4files/package.scala

```diff
@@ -176,12 +176,10 @@ package object solr4files extends DebugEnhancedLogging {
       if (left.getProtocol.toLowerCase == "file")
         Try(new File(left.getPath).length)
       else {
-        Try(http(left.toString).method("HEAD").asString).flatMap {
-          case response if response.isSuccess =>
-            response.headers("content-length").headOption
-              .map(cl => Success(cl.toLong))
-              .getOrElse(Failure(HttpStatusException(s"no content-length header found in response of $left", response)))
-          case response => Failure(HttpStatusException(s"getContentLength($left)", response))
+        val fileSizesUrl = left.toString.replace("bags", "bags/filesizes")
```

Comment:
please implement this using a proper URL, not by replacing it from somewhere. In `Vault` this URL is built for the file; I would like to see a similar URL for the filesize there.

Reply:
I think it is better to pass into this method (`getFileSize`) the "normal" url of the file and then change the url when querying the file size over http. Because it is also possible (anyway it is taken into account in the code) that the protocol is `file`, and then you have to use the "normal" url for the file.

Reply:
* The code can lead to errors if there are multiple instances of the string "bags" in the URL.
* The code doesn't clearly express what is going on, i.e. we are expecting a URL of a [certain pattern](https://dans-knaw.github.io/easy-bag-store/api.html#/default/get_stores__store__bags__uuid___path_) and converting it to another pattern (the new endpoint with "filesizes"). You cannot see this when you read this code.